docs/index.asciidoc

```diff
@@ -84,6 +84,38 @@ like `myapp-%{sequence_id}`. Reusing ids will cause documents to be rewritten.
   * There is no default value
 
 The Engine name. Engine is your search engine created in App Search, an information repository that includes the indexed document records.
+Engine field also support {logstash-ref}/event-dependent-configuration.html#sprintf[sprintf format], to enable
```

Comment:
Should `Engine field` link to the docs for that field?

Reply:
That could be nice, I'll add the link with the suggestions that come from @karenzone

Reply:
Ignore this suggestion - I didn't notice that this was part of the definition for `engine`... 🤦